charts/argo/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: v2.8.0
 description: A Helm chart for Argo Workflows
 name: argo
-version: 0.9.4
+version: 0.10.0
```

Comment:
We normally just bump the patch in the argo chart and bump the minor when pushing a new version (i.e. the argo image version is upgraded)

Reply:
OK, we usually bump our minor versions when changing the UX - i.e. new properties.  Your guidelines say a minor version for new chart functionality - but as you like.  Please let me know if you want this changed or you're just sharing information :)